mne/io/constants.py

```diff
@@ -47,8 +47,8 @@ def __setattr__(self, attr, val):
 FIFF.FIFFB_SMSH_RAW_DATA      = 119
 FIFF.FIFFB_SMSH_ASPECT        = 120
 FIFF.FIFFB_HPI_SUBSYSTEM      = 121
-FIFF.FIFFB_EPOCHS             = 122
-FIFF.FIFFB_ICA                = 123
+FIFF.FIFFB_EPOCHS             = 122   # Before version 0.11 we errantly used this
+FIFF.FIFFB_ICA                = 123   # Before version 0.11 we errantly used this
```

Comment:
Or actually maybe it's better to leave the bad versions out from `constants` entirely, and when you need to look up the "old" version, use the correct constant for the given integer. It will look weird in the code but it makes clear what we were accidentally doing before.